pilot/pkg/features/pilot.go

```diff
@@ -663,6 +663,9 @@ var (
 		"Metric scope rotation interval, set to 0 to disable the metric scope rotation").Get()
 	MetricGracefulDeletionInterval = env.Register("METRIC_GRACEFUL_DELETION_INTERVAL", 5*time.Minute,
 		"Metric expiry graceful deletion interval. No-op if METRIC_ROTATION_INTERVAL is disabled.").Get()
+
+	OptimizedConfigRebuild = env.Register("ENABLE_OPTIMIZED_CONFIG_REBUILD", true,
```

Comment:
Do we want to reuse this flag for other opetimizations later

Reply:
Yes. That is the idea if we implement similar optimization for other configs